emoji-ui/src/components/emojiItem.js

```diff
@@ -4,14 +4,17 @@ import './emojiItem.css';
 
 class EmojiItem extends Component {
   static propTypes = {
-    isSelected: PropTypes.bool.isRequired,
-    url: PropTypes.string.isRequired,
```

Comment:
Why'd you make the props no longer required?

Reply:
i am thinking if an emoji is already imported, maybe they don't need a click handler.

Reply:
in general default always to using isRequired unless you already know it won’t always be sent
you’ll get much better feedback in the console if there are bugs passing the wrong things
and it’s easier to relax a requirement than to remember to enforce an optional one

Reply:
oh @clairewild i missunderstood your comment. yes, the emoji needs to be required. it was a mistake. i will update that.

the clickHandler will remain optional.